### ISSUE TITLE
Rename MovieButton

### ISSUE THREAD

**Issue by mbantwerpes** [marwinburesch/moviescout#36]
Rename movie button to a more appropriate name. Maybe IconButton?

**Comment by marwinburesch** [comment]
I agree, it also returns [an anchor](https://github.com/marwinburesch/moviescout/blob/d4c7035ca00fda6a0c6a7c2b333727dd825a774c/src/app/components/MovieButton/MovieButton.tsx#L33), not a button.... :)
